boltdesignsystem/bolt
# DS-427 Enhance Teaser description ellipsis display

## Jira

https://pegadigitalit.atlassian.net/browse/DS-427

## Summary

Improves the ellipsis display for Teasers that have short description text.

## Details

1. Updated CSS to display ellipsis based on a class
2. Updated Twig to include logic that checks the description content length
3. This is not perfect but should cover 99% of use cases (the other alternative is to use complex JS to count the characters and lines, I don't think it's worth it)

## How to test

Run the branch locally and check the Teaser description on hover doc. Play around with the description content, try different length above and below 80 characters. Make sure it is working properly.
